common-theme/layouts/_default/overview.html

```diff
@@ -23,27 +23,24 @@
       {{ $themePage := . }}
       {{ $relevantBlocksPage := or .Params.blocks_page_for_overview "prep" }}
       {{/* this is the same strategy used on success page. Look there for how to add extra pages */}}
-      {{ $prep := .Site.GetPage (printf "%sprep/index.md"
-        .CurrentSection.RelPermalink)
-      }}
-      {{ with $prep }}
-        {{ if not $hasShownModuleTitle }}
-          <h4><a href="{{ $moduleURL }}">{{ $moduleTitle }}</a></h4>
-          {{ $hasShownModuleTitle = true }}
-        {{ end }}
-        <h5>
-          <a href="{{ $prep.Permalink }}">{{ $themePage.Params.Theme }}</a>
-        </h5>
-        <ol>
-          {{ range .Params.blocks }}
-            {{ $block := ($site.GetPage .src) }}
-            {{ if or $block.Params.hide_from_overview (strings.Contains .src "https:") }}
-              {{ continue }}
-            {{ end }}
-            <li>{{ $block.Title }}</li>
-          {{ end }}
-        </ol>
+      {{ $prep := .Site.GetPage (printf "%sprep/index.md" .CurrentSection.RelPermalink) }}
+      {{ $success := .Site.GetPage (printf "%ssuccess/index.md" .CurrentSection.RelPermalink) }}
+      {{ if not $hasShownModuleTitle }}
```

Comment:
Psst, just from reading, not checking out on local, it looks like you've removed the with. I think you still need with, both with prep and then further down with success. Is it on purpose?

(`[with](https://gohugo.io/functions/go-template/with/)` is the most reliable way of handling null safety in hugo)

Reply:
I just re-pushed this branch as 4 commits - the first three I _think_ are no-op refactors? (Particularly, if you view the third one with "hide whitespace")

I assume removing a `with` block is fine as long as nothing ever tries to use `.` directly? Or does it have some extra nil-related behaviour I don't know about?

Reply:
Well I don't like to print empty blocks on the page, as a rule. I'd prefer to not show anything if there's no content in prep.